annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodDeclarationTree.java

```diff
@@ -46,7 +46,7 @@ public class MethodDeclarationTree extends MetaData<MethodNode> {
   private HashMap<Integer, MethodNode> nodes;
 
   /** Set of all classes flat name declared in module. */
-  private HashSet<String> classNames;
+  private HashMap<String, Set<MethodNode>> classConstructorMap;
```

Comment:
This could also be a `MultiMap` if we use Guava

Reply:
This is also changed to multimap. 2bb3cf428caaa6d3c140106d774a3753bf57f48c